server.js

```diff
@@ -32,8 +41,15 @@ function locationCallBack(req, res) {
 
 app.get('/weather', weatherCallBack);
 function weatherCallBack(req, res) {
-  let weeklyForecast = new getForecast(weatherData);
-  res.send(weeklyForecast);
+  const url = `https://api.weatherbit.io/v2.0/forecast/daily?lat=${req.query.latitude}&lon=${req.query.longitude}&key=${process.env.WEATHER_API_KEY}&days=8
```

Comment:
Your URL is different than mine, I'm pretty sure. I wonder if that's why mine broke....